modules/kn-service-offline-create.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// serverless/cli_reference/kn-offline-services.adoc
+
+[id="creating-an-offline-service_{context}"]
+= Creating an offline service
+
+.Prerequisites
+
+* {ServerlessOperatorName} and Knative Serving are installed on your cluster.
+* You have installed `kn` CLI.
+
+.Procedure
+
+. Create an offline Knative service:
++
+[source,terminal]
+----
+$ kn service create event-display \
+    --image quay.io/openshift-knative/knative-eventing-sources-event-display:latest \
+    --target ./ \
+    --namespace test
+----
++
+[source,terminal]
+----
+Service 'event-display' created in namespace 'test'.
+----
++
+* The `--namespace test` option places the new service in the `test` namespace.
++
+If you do not use `--namespace`, the service is created in the `default` namespace.
++
+* The `--target ./` flag enables the offline mode and specifies `./` as the directory for storing the new directory tree.
++
+If you do not specify an existing directory, but use a filename, such as `--target my-service.yaml`, then no directory tree is created. Instead, only the service descriptor file `my-service.yaml` is created in the current directory.
++
+Additionally, specifying a filename ending in `.json` results into creating the service descriptor file in the JSON format.
+
+. Examine the created directory structure:
++
+[source,terminal]
+----
+$ tree ./
+./
+└── test
+    └── ksvc
+        └── event-display.yaml
+
+2 directories, 1 file
+----
++
+* The old `./` directory specified with `--target` contains the new `test/` directory, named after the specified namespace.
+* The `test/` directory contains the `ksvc` directory, named after the resource type.
+* The `ksvc` directory contains the file `event-display.yaml`, named according to the specified service name.
+
+. Examine the generated service descriptor file:
++
+[source,terminal]
+----
+$ cat test/ksvc/event-display.yaml
+----
++
+[source,terminal]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  name: event-display
+  namespace: test
+spec:
+  template:
+    metadata:
+      annotations:
+        client.knative.dev/user-image: quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+      creationTimestamp: null
+    spec:
+      containers:
+      - image: quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+        name: ""
+        resources: {}
+status: {}
+----
+
+. List information about the new service:
++
+[source,terminal]
+----
+$ kn service describe event-display --target ./ --namespace test
+Name:       event-display
+Namespace:  test
+Age:
+URL:
+
+Revisions:
+
+Conditions:
+  OK TYPE    AGE REASON
+----
++
+* The `--namespace` option specifies the namespace, which communicates to `kn` the subdirectory that contains the necessary service descriptor file.
++
+Omitting the `--namespace` option makes `kn` search for the service in the `default/` subdirectory.
```

Comment:
here it happens the same as suggested by @dsimansk (in create offlinecase), in case you omit the namespace is recognized by the current session you have (in case you have any), or default otherwise.